### ISSUE TITLE
Major changes for 0.4.0

### ISSUE THREAD

**Issue by NillerMedDild** [NillerMedDild/Enigmatica6#528]
Mod Removals:

- [x] Extra Discs (Replaced by ExtraStorage)
- [x] Bad Wither No Cookie Reloaded (Replaced by Extreme Sound Muffler)
- [x] Useful Backpacks (Replaced by Sophisticated Backpacks)
- [x] Ensorcelled (Turns out CoFH Ensorcellation has a similar enchant)
- [x] Biomes O' Plenty (Not needed)
- [x] Fishing Real (Less point without Aquaculture): Removed early.
- [x] Tinkers Mechworks (Content overlap)
- [x] QuantumStorage (Replaced by Industrial Foregoing Black Hole Tanks/Units)
- [x] Rare Ice (It annoys me because it keeps it's config open at all times)
- [x] Angel Ring (Many other ways to fly in a more balanced way)
- [x] Additional Bars? (Doesn't add much uniqueness)
- [x] Dangerous Stonecutter (More annoyance than fun feature)
- [x] Snow Under Trees

Mods to investigate: 
- [x] [BetterEnd Forge](https://www.curseforge.com/minecraft/mc-mods/betterend-forge-port) [#812]

Mod Additions:
- [x] TerraForged

Datapack Changes:
- [x] Move all non-Vanilla Tetra Datapack materials into a `compat` subfolder, to correctly override Tetra's own.

Script Changes:
- [x] remove.js
    - remove section that removes ExtraStorage disc recipes
- [x] Remove outcommented scripts (marked with `//TODO: Remove`

Config Changes:

- [x] Update default keybindings
Lost Trinkets disabling will no longer happen, as the ones we deem too powerful no longer unlock randomly. This allows us to use them as rewards and the like.

**Comment by Thiana** [comment]
How will the removal of Biomes O' Plenty effect existing 0.3.x worlds?

**Comment by NillerMedDild** [comment]
I don't know to be honest, but these changes won't be made until a time where we recommend a world reset anyway.

**Comment by Natalia-Oreiro** [comment]
when updating to version 4.0, will the world be completely erased on the official E6 server?
a good solution would be to keep the chunks that are claimed and remove the others.

**Comment by Merlo17** [comment]
World resets are highly likely in between major updates, since we're going to use them to implement new worldgen or breaking changes. No way around it I'm afraid, claimed chunks would be very broken in new worldgen, especially if terraforged is ready for 0.4
It's the unfortunate nature with beta packs

**Comment by Natalia-Oreiro** [comment]
so will there be a complete reset and everyone will lose everything or no?

**Comment by Merlo17** [comment]
That's very likely, yes

**Comment by Merlo17** [comment]
Tinker's Mechworks should probably also be removed. It only adds drawbridges, covered by modular routers, and firestarters, which can easily be emulated with a fake player

**Comment by NillerMedDild** [comment]
Do Modular Routers work well as drawbridge?

**Comment by TemperedFool** [comment]
I feel like this will be one of a hundred "but couldn't you just..." comments, but drawbridges seem like a task Create was built for?

**Comment by MuteTiefling** [comment]
> Do Modular Routers work well as drawbridge?  

They work very well, actually

**Comment by NillerMedDild** [comment]
Right, true. Bye Mechworks!

**Comment by Thiana** [comment]
My E6 World #3 had lots of cold biomes and Rare ice does generate without issue. Though it's more annoying than anything else since if you accidentally right click anything on ice the item is placed into the ice and you need to break it out.

**Comment by KrAzYGEEK32** [comment]
> My E6 World 3 had lots of cold biomes and Rare ice does generate without issue. Though it's more annoying than anything else since if you accidentally right click anything on ice the item is placed into the ice and you need to break it out.

i believe its normal ice, it just behaves that way due to the mod, basically if any item is clicked onto ice, it will become rare ice and hold the item, atleast that's what i understand.

**Comment by Merlo17** [comment]
Here's a few more suggestions by my part:

- Switch Iron Chests with Expanded Storage (https://www.curseforge.com/minecraft/mc-mods/expanded-storage-forge): this was suggested before on discord and I finally decided to take a look: the mod is really well done and it's honestly an objective upgrade over Iron Chests, so I'd push for the switch. The only concern I have about it blocking it for now is this issue (https://github.com/NinjaPhenix/ForgeMods/issues/11) I reported, so let's see how that turns out before anything. If it's fixed, I'd love to see it included

- Reconsider Flux Networks: another popular one that's asked a lot on discord, and after looking into it I'd include it. The wireless FE stuff is pretty redundant, as we already have other ways to do it in the pack, but we don't have anything that resembles the controller and the amount of control it gives over your power grid. Currently there's nothing to monitor your inputs and outputs, and I've seen many people ask for stuff like that. Especially in upcoming expert, being able to know detailed stats about your power is very useful, and there aren't alternatives in the pack rn

- Extended Crafting (https://www.curseforge.com/minecraft/mc-mods/extended-crafting): not sure what the plans are for expert, but it might come in useful, so I thought I'd at least mention it here. It could probably be added and hidden for now, but I guess we can always put it in later when it's needed. Same could be said for avaritia I guess, even if I'm not a fan of it at all, depends what direction you want to take expert. Might need some discussion.

**Comment by MuteTiefling** [comment]
Flux networks always felt really buggy to me. Have not tried it yet in 1.16, but it was prone to... weirdness even late in 1.12.
ID should fill this niche, honestly. It has all the tools for those willing to build it. I'd imagine Computer Craft can be used to build a decent display as well.

Extended crafting seems rather redundant at this point. Larger crafting grid? Create has it covered. Infusion style crafting? Ars Nouveau and Botania both have mechanics for this. Granted, we don't have an RF variant of any of these. But... is that needed? It feels overdone. Like it's central to basically every large expert pack's endgame crafting. 

JEC, I'd love to see, simply because I love the little mod. It's great prior to RS, and can be helpful even late into the game when trying to figure out more complicated things that don't make sense to run through RS.

**Comment by Merlo17** [comment]
I'd still add flux networks, since those options have an extremely high entry point for the average player. I don't expect them to be able to code a CC power display or even use ID to do that, while the controller is a simple put down and use. I messed with it a bit in creative earlier and it seemed pretty good, but I'd likely have to play with it in survival more to see if it holds up

**Comment by TemperedFool** [comment]
On Flux Networks - my experience with it in 1.12 has been rock solid. You definitely CAN do all of this with ID/IT, and it's actually a bit of a problem from an Expert POV - ID/IT is way too cheap to work with, and crippling things like the Omnidirectional connector to compensate make some interesting wireless setups that you would probably be OK with otherwise more difficult to build.

**Comment by KrAzYGEEK32** [comment]
just mentioning, terraforged got a release recently https://www.curseforge.com/minecraft/mc-mods/terraforged/files/3181431

**Comment by NillerMedDild** [comment]
I will have another look at Flux Networks & JEC

**Comment by MuteTiefling** [comment]
Should we reconsider Botany Pots too? I feel like they were a nice stop gap early on in 1.15 and 1.16 when a lot of options weren't available. But now we've got many more interesting methods to automate farming, including machines like the Insolator and Garden cloches.

**Comment by Merlo17** [comment]
I would be very fine with them gone

**Comment by NillerMedDild** [comment]
That's not a half bad idea.

**Comment by theboo** [comment]
Yeet it... It's a crutch

**Comment by NillerMedDild** [comment]
A crutch? 😄

**Comment by soepie7** [comment]
Why will Aquaculture be removed? Really liked that mod, spent time making an aquarium and everything.

**Comment by CletusMAngo** [comment]
> Why will Aquaculture be removed? Really liked that mod, spent time making an aquarium and everything.

Yeah i agree i really liked aquaculture aswell

**Comment by Thiana** [comment]
I would prefer to keep Botany Pots because they are a nice compact way of keeping decorative blocks/flowers in stock. Building an IF/etc farms for 20+ types of trees (and can anything provide the flowers that BP can?) would rapidly damage server tps. If you want to massively nerf the pot speed to make IF/etc a far preferable alternative for farming large amounts it would be fine, but not completely removing it. Plus the non hopping pots are nice to use as a decorative item.

**Comment by soepie7** [comment]
Yea, only removing hopper pots may be a nice alternative; an easy way to farm early on, but only if you are willing to put in the manual effort for every single crop you harvest.

**Comment by Thiana** [comment]
Insolator seems to be able to farm a lot of the flowers that BP can but not all. And vice-versa. Also the Insolator seems to have more issues with missing outputs, ie Orchid Sapling not giving sticks/apples.

**Comment by soepie7** [comment]
For that argument, we might be able to just add recipes to the Insolator.

**Comment by Thiana** [comment]
True, but I refer back to the pots are pretty in builds the TE block isnt.

**Comment by MuteTiefling** [comment]
> Insolator seems to be able to farm a lot of the flowers that BP can but not all. And vice-versa. Also the Insolator seems to have more issues with missing outputs, ie Orchid Sapling not giving sticks/apples.

This stuff is fixable though. The majority of the plants in the isolator were added by us, in fact. If you see any missing, I'd appreciate a report so we can fix em.

**Comment by Merlo17** [comment]
> Why will Aquaculture be removed? Really liked that mod, spent time making an aquarium and everything.

I'm in favor of at least reconsidering aquaculture. It adds nice character to the world and does pretty much what alex mobs does to the water biomes. I haven't seen them pop up in any spark profiles or laggy situations, and I don't see the mod harming anything in the pack

**Comment by SeaSaltStar** [comment]
> > Why will Aquaculture be removed? Really liked that mod, spent time making an aquarium and everything.
> 
> I'm in favor of at least reconsidering aquaculture. It adds nice character to the world and does pretty much what alex mobs does to the water biomes. I haven't seen them pop up in any spark profiles or laggy situations, and I don't see the mod harming anything in the pack

The only reason I consider it harmful is that it affects squid spawns which are as of now essential for occultism progression.

**Comment by NillerMedDild** [comment]
@SeaSaltStar in 0.3.11 Book and Quill accepts any black dye :)

**Comment by soepie7** [comment]
> @SeaSaltStar in 0.3.11 Book and Quill accepts any black dye :)

Then, are there still reasons left on why to remove AquaCulture 2?

**Comment by NillerMedDild** [comment]
Other than the fact that it adds more entities, which have the greatest lag potential (albeit Aquaculture does not seem to cause lag), and the content is a bit on the random/niche side, no.

Please keep in mind that removals happen to keep the pack light, so it can be run by as many as possible, and on servers too. We cannot just add/keep all the mods.

**Comment by spdkils** [comment]
On pots.... Garden Cloche does it in 1 block just like pots, but eats power....
The other block does it as well in 1 block with power.

But there are 20 other ways to automate a garden. You can use ars, and spells, you can use botania, you can use create, etc etc etc. What is the point of the removal? This is a kitchen sink pack, and there are 40 ways of doing everything. Removing pots just makes the path of least resistance a garden Cloche, and just limits designs and builds.

So, not a fan of removal, personally.

UNLESS it's for server TPS or something, then that makes sense... but, I'm going to build 40 garden cloche, and everything tics.

**Comment by TemperedFool** [comment]
Ignoring the FPS render since most of the alternates have it too, it's just a much cheaper version of the cloche, and the mod does nothing else. All those other mods do lots of stuff that justify their existence. You also need some IE infrastructure to get to cloches; all you need to get hopping botany pots is go for a swim and do a little mining you were already going to do.

**Comment by Kinagh** [comment]
On pots...I do not get why having something that means all you have to do is "go for a swim and do a little mining" is bad.  I play this stuff for relaxation and am not looking for a second job =).  Having an aesthetically pleasing and easy way to grow some stuff placed on pretty looking chests and not needing power is great for those of us who do not have hours and hours to play.

**Comment by TemperedFool** [comment]
Sorry, it wasn't a value judgement, but I was just putting up my interpretation of why Botany Pots is up for discussion. I totally get personally what you're saying - I'm an adult with 2 kids and a full-time job, I definitely get it! I can just see why the discussion is happening on whether or not to keep it.

**Comment by toothmonsterr** [comment]
I wanted to throw in my two cents for keeping Aquaculture: It makes an otherwise really empty part of the world feel very alive. Fishing without it is so monotonous and boring and having the tackle box and actual living fish in the water makes it much more tolerable. I've never had any issue with squid spawning, in fact i get plenty beaching themselves at my base that I don't even have to hunt them down. And the ability to smelt jellyfish into slime is such a lifesaver for the massive amount of recipes that need slime in the pack. I'm not sure what the downsides to it are, honestly, unless it's being replaced with a different aqua-life mod.

**Comment by CletusMAngo** [comment]
An issue with Terraforged that i would like to mention before it is added, Most BYG biomes do not spawn with terraforged i would say a good 60% of biomes dont spawn with terraforged installed
Just thought id let you guys know before you add it

**Comment by Merlo17** [comment]
> On pots...I do not get why having something that means all you have to do is "go for a swim and do a little mining" is bad.  I play this stuff for relaxation and am not looking for a second job =).  Having an aesthetically pleasing and easy way to grow some stuff placed on pretty looking chests and not needing power is great for those of us who do not have hours and hours to play.

On this, I don't really agree. While I get it and it's true, imo it's a little too easy, and it completely kills one aspect/farm you could do. By that logic you could argue that mining takes time, so we should implement a way to constantly farm materials super early and so on. 
Sure, it's a kitchen sink pack where you can do what you want, but it also means that we need to make all options valuable and satisfying. Pots not only break every other farming option in the game, but also aren't satisfying at all, since you can just mass produce them.
I don't think progressing a bit into IE/IF/Thermal/Astral  (and more likely) for a tree farm takes too much time, and the end result is much more satisfying, as it's a whole setup you had to work to achieve. If we had magic block solutions for every item in the game then you might as well play creative

**Comment by Merlo17** [comment]
> An issue with Terraforged that i would like to mention before it is added, Most BYG biomes do not spawn with terraforged i would say a good 60% of biomes dont spawn with terraforged installed
> Just thought id let you guys know before you add it

That can be configured and we're working on it. Also even with default TF touching no configs I had BYG biomes spawning

**Comment by CosmicMermaid** [comment]
Regarding Botany Pots, I'd like to keep the mod. Everyone has a different game-play style. Don't use the pots if you don't like them. Simple. I just got into the pots and I've been playing for over a month. I've been gathering resources all this time and just got tired of it. Saves me time to do other things in the modpack. I got ore-tripling before I started using the pots. Am I missing something with the pots? I'm basically growing trees and crops.

**Comment by Darkere** [comment]
What you are describing is exactly rhe problem. Botany pots overshadows all other methods of obtaining these materials.

Because of botany pots you are now not playing and discovering all the other possible ways you could farm these materials.
 Additionally scaling up production of these materials is incredibly boring when using them (Not to mention pretty ugly).

**Comment by CosmicMermaid** [comment]
Well, I beg to differ. I'm actually using the pots to free up time to explore the modpack more. I'm fairly new to mods so for me it's an intermediary solution. I'm getting into Ars Nouveau and Industrial Foregoing as we speak so I can then transition to those mods and upgrade. Like I said, everyone has different styles of playing. If I didn't have a simple way to automate crops I would have felt slightly overwhelmed. I also like to use a lot of the different woods, flowers, etc to decorate. I know that's not a big thing for folks in playing the mod. I have a whole greenhouse  of pots growing flowers and trees to decorate my base. LOL Also, I'm attaching a couple of pics of my very cute setup which I far from ugly. Ok. I'm done. Thanks! 
![Enigmatica 6 - Minecraft 1 16 5 - Multiplayer (3rd-party Server) 1_31_2021 10_38_07 AM](https://user-images.githubusercontent.com/78312602/106391436-e60ef300-63b2-11eb-8eb1-2ffca3213c09.png)
![Enigmatica 6 - Minecraft 1 16 5 - Multiplayer (3rd-party Server) 1_31_2021 10_54_50 AM](https://user-images.githubusercontent.com/78312602/106391439-e7d8b680-63b2-11eb-9213-67a8c498ee7f.png)

**Comment by msbrittgaming** [comment]
I guess I am not sure I understand why botany pots needs to be removed. People can either choose to use them or not. I am working my way through the mods and would like to spend my limited time on things I care about the most. If botany pots lets me grow sugarcane easy because I need a lot of paper and I CHOOSE that method over a method another mod offers, what's the harm to anyone else not in my game? At the moment I am using various mods to do different things and I love I could choose my preference. It seems the reason for removing the pots is to force people to play a certain way and I thought games like this were meant to allow you to play in the way that suits you? How I choose to get the resource should be up to me. Now if the issue is people are on servers where they feel it is unfair that others can use pots so easily, then that's a personal issue. I for one am on a server with 2 other people and we like the pots because it frees up our time to do other things we enjoy. And if someone is going to play a pack like this with lots of mods and chooses not to get into certain mods because they don't want to, again, that's their choice, who is it hurting? Just my thoughts on why I don't think it should be removed, it seems the reasons are to control how people play which is a little odd, it's a game.

**Comment by TemperedFool** [comment]
I think that is exactly why their removal is being considered. They don't inspire farming creatively. I get that they easily get resources for doing other things - I use them too! But it's also caused me not to bother with other farming methods, because tbh they are too good.

It would make it easier to play with various mods mechanically if you just had creative access to resources. The point of a mod pack that isn't All The Mods is to draw some lines on what content is included, and hopefully balance things a bit. Otherwise we'd all just throw together a jumble of mods and call it a day.

As a thought experiment, and maybe actual suggestion, what if just Hopping Botany pots were removed? They really are the bigger problem from a balance POV; the regular ones still provide a bit of utility early-game, but don't just eliminate the need for other automated farming methods.

**Comment by TemperedFool** [comment]
As a balance suggestion, maybe require a high-end metal for the hopping pots. I was tossing around shadow metal from create, alfsteel, and similar; Boo had a better idea: set up an "endgame material" tag so most mods can get them (and similar things) at their endgames.

**Comment by r-mc2** [comment]
If I can throw my 2 cents in on Botany Pots as well, a lot of it sounds like it is boiling down to players saying
> "We like them because they allow us to define our own personal play style and obtain resources in the way we want to do so."

While the pack authors are saying
> "We don't like them because they don't allow the player to define their own personal play style in the way we want them to define it."

It sounds like your minds are fairly made up already.

Do they make resource gathering easier? Yes. Are they game breakingly OP? Depends on what you're using them for. As an example, try getting a Botany Pot to feed a Bioreactor with the 9 unique plant varieties. They're slow as molasses. You'll have to set up a better farming method if that's your goal; cloches, IF farms, etc. I personally set up an Astral ritual to make plants grow faster + IF machines to harvest and plant to keep up with the machine's demand.
And I feel I should mention it before someone else does, yes, I could have set up a bank of pots to grow more, but that didn't fit my personal choice in play style so I didn't use them and setting up the in-world farm was more fun.

Their existence does not deny the option of doing something else. Removing them does though.

As for them being ugly? Yes, that is purely subjective and not a reason to remove them 😆 (granted I agree they are ugly compared to their inspiration the Bonzai pots lol)

**Comment by MuteTiefling** [comment]
I was thinking earlier that removing hopping pots would probably suffice. That keeps them available for decorative use, but makes them harder to automate.

That said, I've been working on overhauling some scripts over the last few days and some relatively recent changes to kjs make it possible to easily control the rates of pots. Their growth rate could now be easily put on par with standard crops, or slower considering you don't even need light or water nearby.

Output rates are also considerably easier to modify now.

So maybe they stay but get severely nerfed to make other methods more interesting to explore.

**Comment by Merlo17** [comment]
Endgame items certainly aren't the solution: this is still a kitchen sink pack where everything pretty much goes, and gating stuff arbitrarily is even worse than removing imo.
Here's why the argument for "if you don't like it don't use it" doesn't work for me: even if I have more fun with other options that aren't pots, how can I use them knowing that a far cheaper and objectively better alternative exists? 

The problem is all about balance here: they simply are too good and overshadow any other interesting and complex idea other mods have and new players especially that don't know any of these will never feel compelled to explore new ways to farm those materials if they already have a perfect solution in their laps.

Removing the hopping pot works fine for me, since this is an automation problem after all, and even tweaking the rates heavily could work. The point is the more complex and expensive alternatives should feel rewarding and actually provide a significant upgrade. If you sink a ton of resources in an IF farm and it's just slower than three magic pots you just feel like an idiot

**Comment by r-mc2** [comment]
I could see up to a 50% reduction in growth speed on the hopping variants and maybe a 25% reduction on the non-hopping variants. It still gives you some passive generation without having to remember to go harvest it like growing them in-world, but doesn't lead to a full storage drawer overnight (IRL). And the non-hopping will still require setup for automation purposes or if people want speed they can automate the non-hopping or wait til they can earn some higher tier version of material to make better dirt. Although I could see it leading down the path of adding all kinds of dirt types similar to that pack back in 1.12 I can't recall the name of off the top of my head who's early-game was all around the various trees, bonzai pots, and special dirts.

But some options for "better dirt" could be using the fertilizers from CFBH + the tilled farmland variants available with the recipes that use the hoe. I'd be hesitant to think of taking from another mod that isn't food/farming related as it could feel like a "gate" in a non-expert version of the pack, a means to force someone to go down a road they were never gonna touch and risk turning them off the mod entirely if they have to begrudgingly go through it.

But some other options to combine with the tilled farmlands:
* A bucket of biofuel or liquid meat from IF
* Those overgrowth seeds from Botania (or whichever is the non-dungeon loot variant you get from tall grass)
* An Aevitas attuned rock crystal from Astral
* Phytogrow from Thermal
* I'm sure there's something in PNC related to crop growth but I've not gone through the mod myself yet
* Totem of Joy from Nature's Aura (or something more plant related, but happy plants == fast plants, ya?)
* A full jar of mana from Ars (since the jars fill from plants growing nearby)
* A Carbuncle charm from Ars (to imply the carbuncle is automating your crop within the dirt making it faster?)
* A Speed upgrades from any the mods that add them

I imagine we could think of others, but this is a small list for now.

**Comment by Merlo17** [comment]
I'm not sure I see the point of giving the option to combine that stuff with farmland. You came all that way for an interesting and fun treefarm, why would you want to go back to a pot, even if it was faster? It doesn't make sense to me

**Comment by r-mc2** [comment]
> even if I have more fun with other options that aren't pots, how can I use them knowing that a far cheaper and objectively better alternative exists?

Not to be rude, but in the words of another, "That sounds like a you problem" 😄 
Not all of us are gonna be chasing efficiencies and the "best" option. I may fall into that category, but there's also something I enjoy about seeing a variety of things in my worlds so I'd use them for one farm but not for another.

If the concern is teaching new players and worrying that they won't explore other options unless you hold their hand and walk them down the path you want are you really letting them explore and learn? A lot of what I'm also hearing comes across as:
> I'm bored of botany/bonzai pots. They're bland and I want to see something nicer in *everyone's* worlds cause I've done them to death.

But back to these new players, they've never seen a large portion of any of the mods or at least haven't seen something like a Botany Pot, so what's the harm in letting them explore the pots and coming to a similar level of "over it" with them that others have? I feel like removing them denies that experience altogether for the player.

Why go the route of providing better dirt options?
For those players that were never gonna explore Thermal, Create, IF, Ars, or the half-dozen other ways to automate a farm and gives them something to help with output but, the faster options are something you earn later while still giving that gentle nudge of "sure you don't wanna give the IF Plant Sower another look?"

And yes, you could argue that going half-way through a mod to reach, as an example, something like Ars' Carbuncle charm gets you 90% of the way to a farm with them, but what if I don't want to leave my farm up to some RNG pathfinding of Minecraft entities or the server's tickrate? Or what if I don't like Carbuncles in general? They could be "icky" or not what I was looking for once I experimented with them.

In the end I stick to leaving the decision on using them or not to the player, but do like the idea of exploring nerfing their base production rate.

**Comment by Merlo17** [comment]
I like the nerf too, and that's because of what you said here: 

> something I enjoy about seeing a variety of things in my worlds

I'm all for this, but if they keep being so cheap and so much better than everything else you won't see variety, you'll only see botany pots :P 
Ideally I'd want them at a very basic stage where they can give you a bit but they can't reasonably be used for any complex automation and setup, forcing you to explore other options. Then if you want to keep using pots for all your life you can, but you're also pretty stupid. Fair?

**Comment by telemenar** [comment]
In general my interpretation of some of the current goals even in the Kitchen Sink open ended E6 pack are still around automation, and learning mods, even while trying to keep the mod pack from getting too heavy. 

Botany pots are not really an automation to learn. If you know they are there they are a quick cheap way to get infinite wood/food/plants. They are unlikely to be included in the quests, meaning they don’t help the new people either. They are just a cheap alternative to doing something more thematically aligned with the pack goals.

**Comment by MuteTiefling** [comment]
>And yes, you could argue that going half-way through a mod to reach, as an example, something like Ars' Carbuncle charm gets you 90% of the way to a farm with them

That's the thing though, most farming outings are very early game.

Ars nouveau takes a bit of exploring and a tiny amount of raw materials to do a sylph farm.

Pneumaticcraft, you can get a simple harvest drone running immediately after building a pressure chamber (first thing you do in the mod)

Create can do a lot of excellent farms for some pretty basic materials.

Cloches and Insolators take a bit more effort, but that's more because you need to make Rf than anything. The machines are cheap and available very early.

That's just a few of the more interesting ways to farm crops. There are a few more I just don't feel like laying out.

**Comment by r-mc2** [comment]
Well, we keep talking in circles on this:
> Having options and variety are great! Try other things! No! Not like that! * *slaps botany pot out of players' hands* *

Maybe have something in the Welcome chapter of quests with some extra snarky language to say along the lines of:
"Yes, we know these are here, but shouldn't you try something more interesting? There's so many more cool things to look at in the other mods. Try your best to move away from them as soon as possible. You don't wanna be a pothead do ya? 😄"

With some additional info that their production rates are nerfed to ensure they stay early game and/or are less effective options to other mods.

Insofar as making better dirt options for mid-to-late game? Sometimes you just want a "farm in a block" for your crazy contraption and these are perfect for them. Take for example automating Mek's biomass. Instead of having to figure out ender chests and export buses and regulator upgrades and pipes and on and on you can make it nice, neat, and compact with a hopping botany pot. It's just another option for those who want to go that route. No one is locking you into the need of using them. They're just there.

**Comment by Merlo17** [comment]
The point is we're not saying you can't play with something, but every option must be at least a bit balanced in a pack like this, or it invalidates every other option.
I can see the argument for the nerf over removing it which might've been drastic, but right now pots are objectively better than every other alternative, ans that's an issue, it doesn't simply lie in "I like to use them".

**Comment by TemperedFool** [comment]
Can we acknowledge that the pack makers (some of whom you're arguing with) also have their own vision for what they want the pack to be, even if it doesn't jive with how a few people want to play?

I'll also point out that your "you can just not play with them" argument rings hollow, because the corollary is true - if it's not there, you can add it to the pack. I mean, there are people adding AE2, and that's a bit more complicated as an add-on...

Maybe I can approach this from the other side. If botany pots weren't there from the beginning, and were suggested and rejected now for being unbalanced/not what Niller et al intend the pack to be, would there even be much more discussion? Being unable to throw out a mod "people like a lot" just means they will be VERY unlikely to add and try something as they develop the pack (and make no mistake, the pack is super early - they don't even have the worldgen they want yet). That means basically they can only get play test feedback from the small group of people messing with the develop branch.

TLDR; I'm not even invested in seeing pots go, but making this kind of decision out like it's an attack on a play style (time limited style, one I very much have personally) will just stunt the team's ability to make the pack they envision. If they think pots need the nerf/kick hammer, I trust their judgement. I can always download the mod myself.

**Comment by NillerMedDild** [comment]
Thanks for all your feedback on the matter of Botany Pots. I would like to stop the discussion here, as we've made the call:

- Botany Pots will be staying
- They will be made less efficient - Exact values are undetermined at this point.
- The recipe will be made slightly harder, to incoorporate some form of fertilizer and water
- The Hopping recipe will be made slightly harder-er, but not over the top.

**Comment by NillerMedDild** [comment]
Aquaculture is also off the chopping block by the way, the benifts of it's inclusion outweighs the potential small benefits of removal. Thanks for your feedback again folks, without it Aquaculture would have gotten the axe :P

In the future we may look at making discussion threads for decisions like Botany Pots / Aquaculture.

**Comment by r-mc2** [comment]
(just as I was typing our a response lol...)

I can respect that choice. Thank you for letting us discuss it and I hope that, at least for my part, I did not come across as argumentative but trying to be involved in an open debate.

I love the packs you all create here ❤️

**Comment by Kinagh** [comment]
Thanks for even being open to a discussion.

This is a good compromise. And as r-mc2 said, I too love your packs.  In these trying times they are a very welcome escape.

**Comment by Merlo17** [comment]
Rare ice has a quest in adventure, it should be removed

**Comment by OneLemonyBoi** [comment]
Mantle should be kept as it adds stacked hearts.

**Comment by Zaakjin** [comment]
What about removing angel ring? It's very cheap and makes mekanism jetpack useless. Not requires any kind of mana or energy.

**Comment by NillerMedDild** [comment]
We will be removing the Angel Ring, but a similar effect will be available.

**Comment by KrAzYGEEK32** [comment]
Petition to remove "dangerous stonecutter" mod XD, I feel like its sole purpose is to be a PITA for players lol, and there are way better mob farm alternatives to it
. poor mason villagers also keep dying because of it.

**Comment by Merlo17** [comment]
I would also remove snow under trees: it doesn't make much sense that snow gets under trees anyways, usually when it snows it stays on top of them anyways :P

Also you get this

**Comment by soepie7** [comment]
> I would also remove snow under trees: it doesn't make much sense that snow gets under trees anyways, usually when it snows it stays on top of them anyways :P

First of all I would disagree here; wind blows snow under trees, and foliage isn't a waterproof cover, snow goes through. I am looking outside my window right now and am not seeing a snow-free patch under any tree.

If it bugs with that terrain gen mod; that's a different story.

**Comment by Merlo17** [comment]
well my issues are more with how stuff looks in that screenshot, the rest was a bit of reasoning to say even if we have none under trees it wouldn't look so much out of place, or at least less than the lone trees with snow

also as someone who sees lots of snow this time of the year believe me, it does get underneath but you mostly see clearings under trees, or generally less snow anyways. It only increases when snow falls down from the branches and piles up underneath

**Comment by theboo** [comment]
And here I thought the snow under random trees in a snow free biome was a TF thing lol, didn't realize we had a whole ass mod for it.. it might just not work well with TF's temperature/terrain engine tbh so removal might be needed to not get stuff like from that screenshot

**Comment by CletusMAngo** [comment]
I really think some of the Abnormals mods should be revisited cause it adds so much more than v+ content to the game and alot of people seem to love the mods 
and plus the mods are highly configurable so if there are features that dont fit in the pack you guys can just disable them.

**Comment by StaticButton** [comment]
In regard to "Extra Discs (Replaced by ExtraStorage)" 

During the upgrade to 0.4.0 when Extra Discs are going to be removed. Will all items be lost that are stored in any "Extra Discs"
Will players need to remove all items and offload them onto "Refined Storage" discs before upgrading?

**Comment by TemperedFool** [comment]
Items will be lost @StaticButton but this will be a world-breaking update anyway, so you'll need to make a new world.

**Comment by CletusMAngo** [comment]
> I really think some of the Abnormals mods should be revisited cause it adds so much more than v+ content to the game and alot of people seem to love the mods
> and plus the mods are highly configurable so if there are features that dont fit in the pack you guys can just disable them.

To continue on this comment i think all abnormals mods add new content that makes and changes mechanics from the vanilla game seemlessly to list a few

Upgrade aquatic adds new aquatic life that can fight back and that actually creates and ecosystem and not only that it adds a new method to obtain tridents via thrashers which spawn in frozen oceans

Neapolitan Adds new food methods that have new effects that are so much different to simple farming and farmers delight

Endergetic adds a new unique way of mobility via booflos and the booflo vest and a new end biome which adds new building blocks

Savage and ravage adds reworks the whole raiding system and makes it more lively and fun it also reworks creepers (even tho this feature is annoying to some people and it can be disabled)

Buzzier bees adds new ways of gaining effects and making flower forests have meaning it also has integration with resourceful bees and apotheosis

Atmospheric and autumnity add new biomes that have their own very amazing mechanics and they are very unique to the biomes that terraforged and byg add

I rly hope that this has persuaded you guys into adding abnormals in 0.4 or in the future

**Comment by Merlo17** [comment]
Here's my take on the various abnormals mods:

first of all, neapolitan, endergetic, savage and buzzier bees don't really do much for me, I feel like we already have equal or better mods that do the same thing in the pack and they'd just end up feeling like overlap.
Aquatic is cool, the ocean could always use some more content especially now that we have big terraforged ones.
The biome ones are pretty neat and I would like to see them, I feel like they'd integrate pretty well into our new worldgen and some of them have the potential of looking stunning with our new worldgen. On that note, I feel like https://www.curseforge.com/minecraft/mc-mods/bloomful this one is also a cool potential inclusion

They're all fairly small mods which helps to make their case for a few for me: they're realtively harmless and should be pretty easy to just plug into the pack. This is now or never to revisit them since most touch worldgen, and I feel like some could definitively be missed opportunities for cool additions

on another note, https://www.curseforge.com/minecraft/mc-mods/swamp-expansion this one adds cattails for @theboo and could help with some river decoration for our poor TF rivers without PVJ

**Comment by CletusMAngo** [comment]
> Here's my take on the various abnormals mods:
> 
> first of all, neapolitan, endergetic, savage and buzzier bees don't really do much for me, I feel like we already have equal or better mods that do the same thing in the pack and they'd just end up feeling like overlap.
> Aquatic is cool, the ocean could always use some more content especially now that we have big terraforged ones.
> The biome ones are pretty neat and I would like to see them, I feel like they'd integrate pretty well into our new worldgen and some of them have the potential of looking stunning with our new worldgen. On that note, I feel like https://www.curseforge.com/minecraft/mc-mods/bloomful this one is also a cool potential inclusion
> 
> They're all fairly small mods which helps to make their case for a few for me: they're realtively harmless and should be pretty easy to just plug into the pack. This is now or never to revisit them since most touch worldgen, and I feel like some could definitively be missed opportunities for cool additions
> 
> on another note, https://www.curseforge.com/minecraft/mc-mods/swamp-expansion this one adds cattails for @theboo and could help with some river decoration for our poor TF rivers without PVJ
> 
> > **[Attention Required! | Cloudflare](https://www.curseforge.com/minecraft/mc-mods/bloomful)**
> 
> > **[Attention Required! | Cloudflare](https://www.curseforge.com/minecraft/mc-mods/swamp-expansion)**

Well you guys are in luck Swamp expansion and bloomful are merging into one mod called environmental so yeah and also thats fair about the other mods but the only one i recommend that you guys should test is savage and ravage it adds so much more than what meets the eye other than that thank you for looking at my suggestion

**Comment by NillerMedDild** [comment]
Alright alright, more additions :P https://github.com/NillerMedDild/Enigmatica6/issues/1277
